### ISSUE TITLE
Missing table_test.json

### ISSUE THREAD

**Issue by alecervi** [gersteinlab/Struc-Bench#5]
Hi,

I just noticed that `table_test.json` is missing from `data/table` while the corresponding test files are available for HMTL and Latex (e.g. `data/html/html_test.json`). Would it be possible to upload this missing file from the benchmark in the shared drive link? 
Thanks!

**Comment by gag-source** [comment]
> Hi,
> 
> I just noticed that `table_test.json` is missing from `data/table` while the corresponding test files are available for HMTL and Latex (e.g. `data/html/html_test.json`). Would it be possible to upload this missing file from the benchmark in the shared drive link? Thanks!

Hi, have you solved it yet, I also found the missing file

**Comment by Rainlt** [comment]
> > Hi,
> > I just noticed that `table_test.json` is missing from `data/table` while the corresponding test files are available for HMTL and Latex (e.g. `data/html/html_test.json`). Would it be possible to upload this missing file from the benchmark in the shared drive link? Thanks!
> 
> Hi, have you solved it yet, I also found the missing file

That's true. This dataset is not a complete version, and the author seems not willing to fix it. I decide to give up this dataset and search other text2table dataset.
